### PR TITLE
Bugfix/physicals

### PR DESCRIPTION
**Pull request by eineving**

### CODE REVIEW

File: backend/src/main/java/se/chalmers/dat265/group1/WebApi.java [hunk 1]
```diff
@@ -94,6 +109,21 @@ private static void setupDigitalPartsInterface() {
         put(DIGITALPARTS_URL + DIGITALPART_ID_URL, (request, response) -> dpi.updateDigitalPart(gson.fromJson(request.body(), DigitalPart.class)), gson::toJson);
     }
 
+    private static void setupPhysicalInterface() {
```
Comment (chraker) [inline backend/src/main/java/se/chalmers/dat265/group1/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

File: backend/src/main/java/se/chalmers/dat265/group1/WebApi.java [hunk 2]
```diff
@@ -20,25 +22,38 @@
  */
 public class WebApi {
 
-
     private static CustomerAPI ci;
     private static DigitalPartAPI dpi;
     private static MaterialInterface mi;
-    private static PrintingInterface pi;
+    private static PrintingAPI pi;
+    private static PhysicalAPI phi;
```
Comment (chraker) [inline backend/src/main/java/se/chalmers/dat265/group1/WebApi.java]: MINOR Remove the "phi" field and declare it as a local variable in the relevant methods. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)

**Comment by chraker** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by mlonn — APPROVED** [review]
Looks very good

**Pull request merged**